api/controllers/v2/jobs.ts

```diff
@@ -110,11 +110,15 @@ function prepProgressMessage(
   status: string,
   errorReason: string
 ): string {
-  const msg = `Your Job (<${jobUrl}${jobId}|${jobTitle}>) `;
   const env = c.get<string>('env');
+  const deprecationWarning =
+    env === 'prd'
+      ? 'The Autobuilder for staging will be deprecated by Sept. 30th. Please reach out to #ask-docs-platform on questions on how to migrate to Netlify for staging builds.'
+      : '';
+  const msg = `Your Job (<${jobUrl}${jobId}|${jobTitle}>) `;
   switch (status) {
     case 'inQueue':
-      return msg + 'has successfully been added to the ' + env + ' queue.';
+      return msg + `has successfully been added to the ' + env + ' queue. ${deprecationWarning}`;
```

Comment:
The backticks end up adding the `+` as part of the message. We should fix it so that `env` is properly added to the message

Reply:
Gah, thank you for pointing out that sloppy mistake on my end